pkg/gds/service.go

```diff
@@ -114,6 +120,7 @@ type Service struct {
 	admin  *Admin
 	conf   config.Config
 	certs  *sectigo.Sectigo
+	mock   *mock.Server
```

Comment:
The idea here is to have a way to start the mocked Sectigo server when running locally or via integration tests.

Reply:
I understand what the motivation is, but I'd rather not have the `Service` know anything about the mocking, it should just know about the Sectigo client since that's all it uses and we want to make sure there is a distinct separation of concerns. If Testing=True in the conf, Sectigo should go into mocked mode. I'll comment on that shortly, but likely we'll want to remove this.